dac-ansible/README.md

```diff
@@ -2,6 +2,11 @@
 
 Install data-acc demo with ansible. It creates a bunch of OpenStack VMs, then uses ansible to install a development data-acc enviroment.
 
+Get the required git submodules:
+
+    git submodule init
+    git submodule update
+
```

Comment:
So all the deps are installed on line 40/45 with ansible-galaxy, which is ansible's recommended way, as I understand it.